src/Reflection/ReflectionParameter.php

```diff
@@ -126,7 +126,15 @@ public static function createFromNode(
         $param->isVariadic = (bool)$node->variadic;
         $param->isByReference = (bool)$node->byRef;
         $param->parameterIndex = (int)$parameterIndex;
-        $param->typeHint = (new FindTypeFromAst())->__invoke($node->type);
+        $param->typeHint = (new FindTypeFromAst())->__invoke(
+            $node->type,
+            $function->getLocatedSource(),
+            (
+                $function instanceof ReflectionMethod
```

Comment:
yes - if it is not available, it will be an empty string `''`, which is expected by phpdocumentor type resolver